test/test_cli.go

```diff
@@ -189,7 +196,9 @@ func (s *CLISuite) TestPs(t *c.C) {
 	for _, j := range jobs {
 		t.Assert(j, Matches, "echoer")
 	}
-	t.Assert(app.flynn("scale", "echoer=0"), Succeeds)
+	scale := app.flynn("scale", "echoer=0")
+	app.waitFor(ct.JobEvents{"echoer": {"down": 3}})
```

Comment:
Hmm is that an intermittent failure? It would be good to understand the issue, as this check seems redundant.